src/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$color-blue-accent: #00acdc;
+$color-white: #fff;
+$color-darkblue: #060b35;
+$fonts: roboto, sans-serif;
```

Comment:
```suggestion
$fonts: Roboto, sans-serif;
```

Reply:
https://prnt.sc/MqlIt4ezUzzH
it actually prevented me from running npm test since it fails in lint; it used to be Roboto at first but i changed it to lowercase without second quessing it, it fails again as the screenshot states